mw-webapp/src/service/auth/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const USER_IS_AUTH = "userIsAuth";
+export const PATH = "path";
+export const USER_REGISTERED = "true";
+export const USER_UNREGISTERED = "false";
```

Comment:
if it is hard to describe what happens here with just code, try to use comments

Reply:
ok, add comments